Change 'None' to '' in default AtoM config

jrwdunham opened the pull request:
Fixes migration 0032 so that when an AtoM setting has no value in the previous state of the database it will be set to the empty string instead of `str(None)`, i.e., 'None'.

Fixes #879.

sevein reviewed (changes requested):
Could you make a new migration instead? 0032 is already used in production in the JiscRDSS fork.

jrwdunham committed:
Add migration to fix "None" AtoM config issue

Fixes an issue introduced by migration 0032 wherein an AtoM setting that did
not exist in the previous state of the database was inadvertently created with
a value of 'None'. The migration introduced here changes those 'None's
to ''s.

Fixes #879.

jrwdunham commented:
@sevein I went ahead and made this into a migration. Any objections to merging?

jrwdunham commented:
Cherry-picked to stable/1.7.x as a9e836d62e19e8d3e65a8077334e01c97b377b09.
